alexandria/storages/fields.py

```diff
@@ -10,14 +14,44 @@
 class DynamicStorageFieldFile(FieldFile):
     def __init__(self, instance, field, name):
         super().__init__(instance, field, name)
-        self.storage = storages.create_storage(
-            {"BACKEND": settings.ALEXANDRIA_FILE_STORAGE}
-        )
+        storage_backend = settings.ALEXANDRIA_FILE_STORAGE
         if settings.ALEXANDRIA_ENABLE_AT_REST_ENCRYPTION:
             from alexandria.core.models import File
 
             if instance.encryption_status == File.EncryptionStatus.SSEC_GLOBAL_KEY:
-                self.storage = SsecGlobalS3Storage()
+                storage_backend = "alexandria.storages.backends.s3.SsecGlobalS3Storage"
+        self.storage = storages.create_storage({"BACKEND": storage_backend})
+
+    def copy(self, target_name: str):
+        # S3 compatible storage: copy in same bucket with s3 copy
```

Comment:
```suggestion
        ``` Copy file content to target.
        
        NOTE: this will copy the content and this copy will be the new reference of the filefield. This means
        that there will be no pointer to the original content, unless you made a copy of the parent model 
        instance beforhands by e. g. setting `file_instance.pk` to `None` before calling
        `file_instance.content.copy`.
        ```
        # S3 compatible storage: copy in same bucket with s3 copy
```